resources/views/admin/imet/v2/modules_iterate.blade.php

```diff
@@ -0,0 +1,93 @@
+<?php
```

Comment:
Can you please move the views in a dedicated subfolder like resources/views/admin/imet/v2/tools ?